library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -73,8 +78,8 @@ public void run() {
         migrationStatus.markAsMigrating();
         onUpdate(migrationStatus);
 
-        migratePartialDownloads(migrationStatus, database, partialMigrations, downloadsPersistence, basePath);
-        migrateCompleteDownloads(migrationStatus, database, completeMigrations, downloadsPersistence, basePath);
+        migrateCompleteDownloads(migrationStatus, database, completeMigrations, downloadsPersistence, filePersistence);
```

Comment:
`Complete` downloads need to be done first because otherwise we remove the files if they are shared by the partial download 😬